src/config/index.js

```diff
@@ -0,0 +1,10 @@
+import packageInfo from "../../package.json";
+import language from "./language.json";
+import settings from "./settings.json";
+import config from "./env.js";
+
+export const taskName = packageInfo.name;
+export const taskVersion = packageInfo.version;
+export const LANGUAGE = language;
+export const SETTINGS = settings;
+export const CONFIG = config;
```

Comment:
Could you also add the comments and TODOs from `main.js` into this file as well? You can see them if you look at the deleted code in the PR